tests/unit/tests/add_new_test_file.py

```diff
@@ -5,15 +5,25 @@
 # SPDX-License-Identifier: BSD-3-Clause-Clear
 #
 
-import commands
+import subprocess
 import sys
 import os
 
+def run_command(args):
+    result = subprocess.run(" ".join(args), shell=True,
+                            stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    result.stdout = result.stdout.decode("ASCII", errors='ignore')
+    result.stderr = result.stderr.decode("ASCII", errors='ignore')
+    return result
+
 args = ' '.join(sys.argv[1:])
 script_path = os.path.dirname(os.path.realpath(__file__))
 framework_script_path = os.path.join(script_path, "../framework/add_new_test_file.py")
 framework_script_path = os.path.normpath(framework_script_path)
-status, output = commands.getstatusoutput(framework_script_path + " " + args)
+result = subprocess.run(framework_script_path + " " + args, shell=True,
```

Comment:
Did you mean
result = run_command(......)
?

Reply:
Oops, I've copy-pasted to much. Fixed.